pages/vi/profiles/Going-Gone.md

```diff
@@ -0,0 +1,26 @@
+# Daren Sell
+
+![alt text](https://images-ext-2.discordapp.net/external/CEZuWLsRIiHvgEXmkzFszvzjmdiRrrzzrTMhkVXqbgA/https/i.gyazo.com/thumb/1200/8279b3bc404a283f3ac4100e5a785054-png.jpg "My Drag Racing Car")
```

Comment:
[There was a non-empty alt attribute in your HTML version](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1548/commits/8e27db31ea12a2d4fd74379887fb9ad0d78f4958), why not include it in Markdown as well?

Reply:
I updated it to include the alt text from the prior version.